src/Microsoft.OpenApi/Models/OpenApiCallback.cs

```diff
@@ -34,6 +34,20 @@ public class OpenApiCallback : IOpenApiSerializable, IOpenApiReferenceable, IOpe
         /// This object MAY be extended with Specification Extensions.
         /// </summary>
         public IDictionary<string, IOpenApiExtension> Extensions { get; set; } = new Dictionary<string, IOpenApiExtension>();
+        
+        /// <summary>
+        /// Initializes a copy of an <see cref="OpenApiCallback"/> object
+        /// </summary>
+        public object Clone()
```

Comment:
the interface implementation needs to be declared (: ICloneable in the class declaration) for all models